src/r3/atomic-swap/corda/README.md

```diff
@@ -1 +1,91 @@
-# interop-atomicswap-corda
+# Corda-EVM interop
+
+## License
+
+This project is licensed under the Apache License 2.0. See the `LICENSE` file for details.
+
+## Development Status
+
+The atomic swap reference code is currently under development but nearing completion. Some of its components are in the process of refinement and preparation for community access.
+
+### Component Overview and Status
+
+The following is a list of components included in this project. All of these are subject to future changes and are currently under development.
+
+1. **EVM Interoperability Service**: This service enables flows to execute asynchronous EVM transactions and calls, waiting for the result. It is fully implemented and tested. 
```

Comment:
is there a mapping from these components to specific files or subdirectories?  If so, include a link so the reader can navigate to the code in question?